resources/views/messages/show.blade.php

```diff
@@ -26,8 +26,8 @@
                             <tr class="table__row">
                                 <td class="table__cell">{{ $message->label or 'N/A'}}</td>
                                 <td class="table__cell">{{ $message->subject or 'N/A'}}</td>
-                                <td class="table__cell">{{ str_limit($message->body, 100)}}</td>
-                                <td class="table__cell"><a href="{{ route('messages.send', ['message' => $message->id, 'contest_id' => $message->contest_id]) }}" class="button -tertiary ">Send</a></td>
+                                <td class="table__cell">{{ str_limit($message->body, 100) }}</td>
```

Comment:
should we link to an individual view of a message in case the admin wants to view the full message?